.github/pr-labeler.yml

```diff
@@ -1,3 +1,3 @@
 master: ['hotfix/*', 'release/*', 'dev/*', 'dev*']
 external-dependency: ['combined/*', 'combined-dependabots/*']
-brightlayer-ui: ['*', '*/*']
+brightlayer-ui: ['feature/*', 'bug/*', 'fix/*', 'experiment/*']
```

Comment:
Does this mean that dependabot PRs would never get tagged with `brightlayer-ui`? What about [external forked PRs](https://github.com/etn-ccis/blui-figma-plugins/pull/56) that do not like to follow our own branch naming conventions?

Reply:
dependabot PR gets labels from here.

![image](https://user-images.githubusercontent.com/119693939/233630840-fa3953a1-eb94-46b4-818c-68fa76627204.png)

forks get label from the 'labeler' not pr-labeler see this PR
https://github.com/etn-ccis/blui-react-design-patterns/pull/271

Reply:
I didn't know that, thanks! 

Will dev -> master be properly tagged too?

Reply:
we don't have tagging in this repo as of now